xml/ha_concepts.xml

```diff
@@ -607,122 +607,99 @@
+        class="daemon">pacemaker-execd</systemitem> daemon. Through this daemon,
+        Pacemaker can start, stop  and monitor resources or fence nodes.
+      </para>
       </listitem>
      </varlistentry>
+      <varlistentry  xml:id="vle.cib">
+       <term>Cluster Information Database (CIB)</term>
+       <listitem>
+        <para>
+         On every node, Pacemaker maintains the cluster information database
+         (CIB). It is an XML representation of the cluster configuration
+         (including cluster options, nodes, resources, constraints and the
+         relationship to each other). The CIB also reflects the current cluster
+         status. There is one master CIB in the cluster, hosted on the node that
+         has been elected as Designated Coordinator (DC). From the DC, the CIB is
+         synchronized to the other cluster nodes, which contain a CIB replica
+         each. The <systemitem class="daemon">pacemaker-based</systemitem>
+         daemon takes care of reading and writing cluster configuration and
+         status.</para>
+       </listitem>
+      </varlistentry>
      <varlistentry xml:id="vle.dc">
       <term>Designated Coordinator (DC)</term>
       <listitem>
        <para>
-        One CRM in the cluster is elected as DC. The DC is the only entity
+        The DC is elected from all nodes in the cluster after a membership change
+        (nodes joining or leaving the cluster). The DC is the only entity
         in the cluster that can decide that a cluster-wide change needs to
-        be performed, such as fencing a node or moving resources around. The
-        DC is also the node where the master copy of the CIB is kept. All
+        be performed, such as fencing a node or moving resources around. All
         other nodes get their configuration and resource allocation
-        information from the current DC. The DC is elected from all nodes in
-        the cluster after a membership change.
+        information from the current DC.
        </para>
       </listitem>
      </varlistentry>
-     <varlistentry xml:id="vle.pe.and.te">
-      <term>Policy Engine (PE)</term>
+     <varlistentry xml:id="vle.pe">
+      <term>Policy Engine</term>
       <listitem>
        <para>
-        Whenever the Designated Coordinator needs to make a cluster-wide
-        change (react to a new CIB), the Policy Engine calculates the next
-        state of the cluster based on the current state and configuration.
-        The PE also produces a transition graph containing a list of
-        (resource) actions and dependencies to achieve the next cluster
-        state. The PE always runs on the DC.
+        The policy engine always runs on the DC. The engine is implemented as
+        <systemitem class="daemon">pacemaker-schedulerd</systemitem> daemon. It
+        schedules all actions that are needed whenever the CIB changes
+        changes and calculates the next state of the cluster,
```

Comment:
Redundant "changes" :-)  Actually not necessarily for any CIB changes...

How about something like:
"When cluster transition is needed based on the current state and configuration, pacemaker-schedulerd calculates the expected next state of the cluster , and determines what actions to be scheduled to achieve that. "
?

Reply:
Thanks, changed according to your proposal.